warehouse/templates/email/removed-as-collaborator.subject.txt

```diff
@@ -0,0 +1 @@
+Role removed on PyPI for project {{ project }}
```

Comment:
We're now prefacing all the subjects with something like `[PyPI]` or `[TestPyPI]` so we shouldn't refer to the site name with "on PyPI" in the subject here and in other subjects.